File: backend/backend/Dtos/VetDtos/AnimalDtos/UpdateAnimalVetDto.cs
namespace backend.Dtos.VetDtos.AnimalDtos
{
    public class UpdateAnimalVetDto
    {
        public int Age { get; set; }
        public string Allergies { get; set; } = string.Empty;
        public string AntecedentsMedicaux { get; set; } = string.Empty ;
    }
}

File: frontend/veterinary_app/lib/models/animal_models/update_animal_vet_dto.dart
// veterinary_app/models/animal_models/update_animal_vet_dto.dart

class UpdateAnimalVetDto {
  final int age;
  final String allergies;
  final String antecedentsMedicaux;

  UpdateAnimalVetDto({
    required this.age,
    required this.allergies,
    required this.antecedentsMedicaux,
  });

  Map<String, dynamic> toJson() {
    return {
      'age': age,
      'allergies': allergies,
      'antecedentsMedicaux': antecedentsMedicaux,
    };
  }
}